docs/releases/v2022.1.1.rst

```diff
@@ -0,0 +1,64 @@
+Gluon 2022.1
+============
+
+Important notes
+---------------
+
+This release mitigates multiple flaws in the Linux wireless stack causing RCE as well as DoS vulnerabilities.
+
+
+Added hardware support
+----------------------
+
+ath79-nand
+~~~~~~~~~~~~~~
+
+-  ZTE
+
+   -  MF281
```

Comment:
```suggestion
```

The device is still marked as broken. As such it is not yet supported, right?

https://github.com/freifunk-gluon/gluon/pull/2637/files#diff-cdff5087c568afbf34e2d9c56309ad79c06ddb976f97a04b0c206e34d6065990R37